### DISCUSSION TITLE
SUPublicEDKey missing in project, still works

### DISCUSSION THREAD

**Discussion by jontelang** [sparkle-project/Sparkle#2350] [Q&A] [answered]
Hello. I was reading through the documentation and noticed that I do not have `SUPublicEDKey` inside my app project. It still works to update though, so I wonder what is the difference in adding it / leaving it.

**Comment by zorgiepoo** [comment, marked as answer]
The difference is somewhat hinted in the [Segue for security concerns](https://sparkle-project.org/documentation/#3-segue-for-security-concerns) part of the documentation: using EdDSA gives better security in verifying the download was how you distributed it (the code signing checks are shallow for integrity for various reasons), it has a fail-safe allowing rotation of signing keys in case your developer ID changes (this can make development for certain projects easier too since a development build may use a different certificate or not use one), and it is required if you want to deliver smaller / more efficient binary delta updates. Only using Developer ID code signing is supported for compatibility reasons but not strongly recommended.

**Reply by jontelang** [reply]
Appreciate the answer, thank you.
